blob/drivertest/drivertest.go

```diff
@@ -2215,42 +2224,49 @@ func testSignedURL(t *testing.T, newHarness HarnessMaker) {
 	// PUT the blob. Try with all URLs, only putURL should work when given the
 	// content type used in the signature.
 	type signedURLTest struct {
-		urlMethod   string
-		contentType string
-		url         string
-		wantSuccess bool
+		urlDescription string
+		url            string
+		contentType    string
+		wantSuccess    bool
 	}
 	tests := []signedURLTest{
-		{http.MethodGet, "", getURL, false},
-		{http.MethodDelete, "", deleteURL, false},
+		{"getURL", getURL, "", false},
+		{"deleteURL", deleteURL, "", false},
 	}
 	if putURLWithContentType != "" {
-		tests = append(tests, signedURLTest{http.MethodPut, allowedContentType, putURLWithContentType, true})
-		tests = append(tests, signedURLTest{http.MethodPut, differentContentType, putURLWithContentType, false})
-		tests = append(tests, signedURLTest{http.MethodPut, "", putURLWithContentType, false})
+		tests = append(tests, signedURLTest{"putURLWithContentType", putURLWithContentType, allowedContentType, true})
+		tests = append(tests, signedURLTest{"putURLWithContentType", putURLWithContentType, differentContentType, false})
+		tests = append(tests, signedURLTest{"putURLWithContentType", putURLWithContentType, "", false})
 	}
+	/*
+	*/
 	if putURLEnforcedAbsentContentType != "" {
-		tests = append(tests, signedURLTest{http.MethodPut, "", putURLWithoutContentType, true})
-		tests = append(tests, signedURLTest{http.MethodPut, differentContentType, putURLWithoutContentType, false})
+		tests = append(tests, signedURLTest{"putURLEnforcedAbsentContentType", putURLEnforcedAbsentContentType, "", true})
```

Comment:
This was an actual bug in the test -- it was using putURLWithoutContentType instead of putURLEnforcedContentType.

The rest of the changes here are cosmetic to make it easier to see which test failed.

Reply:
Thank you for doing this! I spent quite a bit of time last night trying to correlate the tests with the failures in Travis.

Reply:
Yea, me too :-).